docs/introduction.md

```diff
@@ -0,0 +1,37 @@
+# Ansible Development Tools (ADT)
+
+## Introduction
+
+Ansible Development Tools or ADT for short, aims to streamline the setup and usage of several tools needed in order to create [Ansible] content.
+When it comes to create automation content using Ansible, there are several packages available that can help users in different parts of the content creating journey. From bootstrapping new projects, all the way to ensuring content follows best practices and verifying it behaves as intended via well established test frameworks.
+
+## Key Features
+
+- All-in-One Ansible Toolkit: ADT combines critical Ansible development packages into a unified Python package.
+
+- Simplified Ansible Automation: With ADT, focus on crafting your automation scenarios instead of dealing with the intricacies of managing and integrating different Ansible libraries.
+
+- Boosted Development Efficiency: By bundling essential Ansible packages, ADT is designed to accelerate development speed and reduce boilerplate code. Spend less time setting up and more time creating powerful automation workflows.
```

Comment:
can we combine points (2) `Simplified Ansible Automation` and (3) `Boosted Development Efficiency` as I think it is trying to convey more or less the same thing.

```
ADT focuses on crafting your automation scenarios and
workflows with speed by reducing boilerplate code without
dealing with the intricacies of managing and integrating
different Ansible libraries
```

Reply:
@ajinkyau sure, done